### PR TITLE
Okta: Okta as an Identity Provider

### PR DESCRIPTION
**Pull request by dallasread**
The goal of this PR is to create the configuration guide that is required to submit our app to the Okta marketplace.

Belongs to https://github.com/dnsimple/dnsimple-business/issues/1661

### CODE REVIEW

File: content/articles/okta-identity-provider.markdown [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+---
+
+Using Okta as an identity provider streamlines the login experience for you and your team.
+
+## Prequisites
+
+In order to proceed with configuring login with SSO through Okta, you must:
+
+- Have administrator access to an Okta tenant
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:23]:
```suggestion
- Have administrator access to an Okta organization
```

File: content/articles/okta-identity-provider.markdown [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+- Have administrator access to a DNSimple account
+- Have the Okta integration feature enabled on your DNSimple account
+
+## Supported Features
+
+- **IdP-initiated SSO**: This authentication flow occurs when the user attempts to log in to DNSimple from Okta.
+- **Import Users** & **Import Groups**: The members in your DNSimple account will stay in sync with the users that you add via the DNSimple Okta app.
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:30]:
```suggestion
- **Import Users** & **Import Groups**: Manage who can access your DNSimple account via Okta
```

File: content/articles/okta-identity-provider.markdown [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+- **SLO (Single Log Out)**: Users can log out of their DNSimple session by logging out of their Okta session.
+
+## Configure Okta with DNSimple
+
+...
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:35]:
```suggestion
Follow the instructions below to connect DNSimple to your Okta organization.
```

File: content/articles/okta-identity-provider.markdown (hunk 2 above)
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:30]:
```suggestion
- **Import Users** & **Import Groups**: Manage who can access your DNSimple account via Okta
```

File: content/articles/okta-identity-provider.markdown [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+### Configure Okta for SSO in DNSimple
+
+To add SSO to your DNSimple account, send an email to support@dnsimple.com with the following information:
+
+- Client ID
+- Client Secret
+- Issuer URL
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:49]:
```suggestion
- Issuer URL (Okta Domain)
```

File: content/articles/okta-identity-provider.markdown [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+## Configure Okta with DNSimple
+
+...
+
+### Add the DNSimple App to Okta
+
+![Linking an Okta Workspace](/files/okta-sso-enable.png)
+
+...
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:41]:
```suggestion
1. Login to your organization's Okta marketplace.
1. Search for DNSimple, and then click Add.
1. Enter an Application Label in General Settings.
1. Click Done.
1. Take note of your Client ID, Client Secret, and Okta Domain as you'll need it in the next step.
1. Before leaving the Sign On tab, fill in the SubDomain field with the Cerby created workspace name, ensuring that all letters are lowercase and all spaces are replaced with dashes.
```

File: content/articles/okta-identity-provider.markdown [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+### Configure Okta for SSO in DNSimple
+
+To add SSO to your DNSimple account, send an email to support@dnsimple.com with the following information:
+
+- Client ID
+- Client Secret
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:48]:
```suggestion
- Okta Client ID (from the previous step)
- Okta Client Secret (from the previous step)
```

File: content/articles/okta-identity-provider.markdown [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+Follow the instructions below to connect DNSimple to your Okta organization.
+
+### Add the DNSimple App to Okta
+
+1. Login to your organization's Okta marketplace.
+1. Search for DNSimple, and then click Add.
+1. Enter an Application Label in General Settings.
+1. Click Done.
+1. Take note of your Client ID, Client Secret, and Okta Domain as you'll need it in the next step.
+1. Before leaving the Sign On tab, fill in the SubDomain field with the Cerby created workspace name, ensuring that all letters are lowercase and all spaces are replaced with dashes.
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:44]:
```suggestion
```

File: content/articles/okta-identity-provider.markdown [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+## Remove the configuration
+
+![Unlink an Okta Workspace](/files/okta-sso-enabled.png)
+
+<div class="section-steps" markdown="1">
+#### To unlink Okta
+
+1. Go to the <label>Account</label> page, and click the <label>Single Sign-On</label> tab.
+1. Click <label>Revoke</label> next to the SSO Workspace you wish to remove.
+</div>
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:83]:
```suggestion
1. Go to the <label>Account</label> page, and click the <label>Single Sign-On</label> tab.
1. Click <label>Revoke</label> next to the SSO Workspace you wish to remove.
```

File: content/articles/okta-identity-provider.markdown [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+
+In order to proceed with configuring login with SSO through Okta, you must:
+
+- Have administrator access to an Okta organization
+- Have administrator access to a DNSimple account
+- Have the Okta integration feature enabled on your DNSimple account
+
+## Supported Features
+
+- **IdP-initiated SSO**: This authentication flow occurs when the user attempts to log in to DNSimple from Okta.
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:29]:
```suggestion
- **IdP-initiated Single Sign-On (SSO)**: This authentication flow occurs when the user attempts to log in to DNSimple from Okta.
```

File: content/articles/okta-identity-provider.markdown [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+- **Import Users** & **Import Groups**: Manage who can access your DNSimple account via Okta
+- **SLO (Single Log Out)**: Users can log out of their DNSimple session by logging out of their Okta session.
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:31]:
```suggestion
- **Single Log Out (SLO)**: Users can log out of their DNSimple session by logging out of their Okta session.
```

File: content/articles/okta-identity-provider.markdown [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+
+If the Okta app is no longer linked (e.g. the access token is revoked), you can re-link the Workspace using the "Re-connect" button. -->
+
+### Assign People or Groups
+
+To give people or groups access to DNSimple, click "Assignments" in under the configured DNSimple app in your Okta dashboard. Then, click "Assign" and select the appropriate users or groups. Assigned users (or users via groups) will now be able to log into DNSimple via SSO.
+
+<info>
+Our integration works with groups! Leverage them to simplify your team access management.
+</info>
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:70]:
```suggestion
<info>
Our integration works with groups! Leverage them to simplify your team access management within Okta.
</info>

### Access control

Now, you can set the level of DNSimple access for each member by visiting the Account > Members tab in your DNSimple account. For the full details, visit [Domain Access Control](/articles/domain-access-control/)
```

File: content/articles/okta-identity-provider.markdown [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+
+To give people or groups access to DNSimple, click "Assignments" in under the configured DNSimple app in your Okta dashboard. Then, click "Assign" and select the appropriate users or groups. Assigned users (or users via groups) will now be able to log into DNSimple via SSO.
+
+<info>
+Our integration works with groups! Leverage them to simplify your team access management within Okta.
+</info>
+
+### Access control
+
+Now, you can set the level of DNSimple access for each member by visiting the Account > Members tab in your DNSimple account. For the full details, visit [Domain Access Control](/articles/domain-access-control/)
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:74]:
```suggestion
Now, you can set the level of DNSimple access for each member by visiting the Account > Members tab in your DNSimple account. For the full details, visit [Domain Access Control](/articles/domain-access-control/).
```

File: content/articles/okta-identity-provider.markdown [hunk 13]
```diff
@@ -0,0 +1,81 @@
...
+## Configure Okta with DNSimple
+
+Follow the instructions below to connect DNSimple to your Okta organization.
+
+### Add the DNSimple App to Okta
+
+1. Login to your organization's Okta marketplace.
+1. Search for DNSimple, and then click Add.
+1. Enter an Application Label in General Settings.
+1. Click Done.
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:42]:
```suggestion
1. Click Done.
1. Assign yourself to the application.
```

File: content/articles/okta-identity-provider.markdown [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
+
+Once we receive that info, we can enable the feature on your account and add the Okta configuration.
+
+<!-- 1. Go to the <label>Account</label> page, and click the <label>Single Sign-On</label> tab.
+1. Clicking <label>Add</label> next to the Okta SSO provider will take you to Okta to authorize the Workspace. You'll need to be logged-in to the Okta Workspace as an administrator of the workspace.
+1. Okta will show you what access DNSimple is requesting.
+1. Okta SSO is enabled on the account.
+1. Repeat the process for each account where you want Okta SSO enabled.
+
+If the Okta app is no longer linked (e.g. the access token is revoked), you can re-link the Workspace using the "Re-connect" button. -->
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:63]:
```suggestion
1. Go to the <label>Account</label> page, and click the <label>Single Sign-On</label> tab.
1. Click <label>Authorize with Okta</label> to authorize the Workspace. You'll need to log in to the Okta Workspace with access to the `okta.eventHooks.manage`  and `okta.users.read` scopes.
1. After logging in via Okta, you will arrive back on the DNSimple Single Sign-On page with Okta SSO enabled for the account.
1. Repeat the process for each account where you want Okta SSO enabled.

<!-- If the Okta app is no longer linked (e.g. the access token is revoked), you can re-link the Okta application using the "Reauthorize with Okta" button. -->
```

File: content/articles/okta-identity-provider.markdown [hunk 15]
```diff
@@ -0,0 +1,72 @@
...
+1. Go to the <label>Account</label> page, and click the <label>Single Sign-On</label> tab.
+1. Click <label>Authorize with Okta</label> to authorize the Workspace. You'll need to log in to the Okta Workspace with access to the `okta.eventHooks.manage`  and `okta.users.read` scopes.
+1. After logging in via Okta, you will arrive back on the DNSimple Single Sign-On page with Okta SSO enabled for the account.
+1. Repeat the process for each account where you want Okta SSO enabled.
+
+<!-- If the Okta app is no longer linked (e.g. the access token is revoked), you can re-link the Okta application using the "Reauthorize with Okta" button. -->
+
+### Assign People or Groups
+
+To give people or groups access to DNSimple, click "Assignments" in under the configured DNSimple app in your Okta dashboard. Then, click "Assign" and select the appropriate users or groups. Assigned users (or users via groups) will now be able to log into DNSimple via SSO.
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:57]:
```suggestion
To give people or groups access to DNSimple, click "Assignments" in under the configured DNSimple app in your Okta dashboard. Then, click "Assign" and select the appropriate users or groups. If an assigned user does not already have a matching DNSimple user with the same email address, a DNSimple user will be provisioned for them and the DNSimple account admin will receive a notification that they have been added. If an assigned user already exists, they will need to link their Okta user on the User Settings page within DNSimple.
```

File: content/articles/okta-identity-provider.markdown [hunk 16]
```diff
@@ -0,0 +1,72 @@
...
+
+- Have administrator access to an Okta organization
+- Have administrator access to a DNSimple account
+- Have the Okta integration feature enabled on your DNSimple account
+
+## Supported Features
+
+- **IdP-initiated Single Sign-On (SSO)**: This authentication flow occurs when the user attempts to log in to DNSimple from Okta.
+- **Import Users** & **Import Groups**: Manage who can access your DNSimple account via Okta
+- **Single Log Out (SLO)**: Users can log out of their DNSimple session by logging out of their Okta session.
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:31]:
```suggestion
## Supported Features

- **SP-initiated Single Sign-On (SSO)**: This authentication flow occurs when the user attempts to log in to DNSimple from Okta.
- **Import Users**: Manage who can access your DNSimple account by assigning users to the Okta application
- **Single Log Out (SLO)**: Users can log out of their DNSimple session by logging out of their Okta session.
```

File: content/articles/okta-identity-provider.markdown [hunk 17]
```diff
@@ -0,0 +1,72 @@
...
+1. Login to your organization's Okta marketplace.
+1. Search for DNSimple, and then click Add.
+1. Enter an Application Label in General Settings.
+1. Click Done.
+1. Assign yourself to the application.
+1. Take note of your Client ID, Client Secret, and Okta Domain as you'll need it in the next step.
+
+### Configure Okta for SSO in DNSimple
+
+1. Go to the <label>Account</label> page, and click the <label>Single Sign-On</label> tab.
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:48]:
```suggestion
1. At DNSimple, go to the <label>Account</label> page, and click the <label>Single Sign-On</label> tab.
```

File: content/articles/okta-identity-provider.markdown [hunk 18]
```diff
@@ -0,0 +1,72 @@
...
+1. Click <label>Authorize with Okta</label> to authorize the Workspace. You'll need to log in to the Okta Workspace with access to the `okta.eventHooks.manage`  and `okta.users.read` scopes.
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:49]:
```suggestion
1. Click <label>Authorize with Okta</label> to authorize the Workspace. You'll need to log in to the Okta Workspace with access to the `okta.eventHooks.manage`  and `okta.eventHooks.read` scopes.
```

File: content/articles/okta-identity-provider.markdown [hunk 19]
```diff
@@ -0,0 +1,72 @@
...
+1. After logging in via Okta, you will arrive back on the DNSimple Single Sign-On page with Okta SSO enabled for the account.
+1. Repeat the process for each account where you want Okta SSO enabled.
+
+<!-- If the Okta app is no longer linked (e.g. the access token is revoked), you can re-link the Okta application using the "Reauthorize with Okta" button. -->
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:53]:
```suggestion
If the Okta app is no longer linked (e.g. the access token is revoked), you can re-link the Okta application using the "Reauthorize with Okta" button.
```

File: content/articles/okta-identity-provider.markdown [hunk 20]
```diff
@@ -0,0 +1,72 @@
...
+
+### Assign People or Groups
+
+To give people or groups access to DNSimple, click "Assignments" in under the configured DNSimple app in your Okta dashboard. Then, click "Assign" and select the appropriate users or groups. If an assigned user does not already have a matching DNSimple user with the same email address, a DNSimple user will be provisioned for them and the DNSimple account admin will receive a notification that they have been added. If an assigned user already exists, they will need to link their Okta user on the User Settings page within DNSimple.
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:57]:
```suggestion
To give people access to DNSimple, click "Assignments" in under the configured DNSimple app in your Okta dashboard. Then, click "Assign" and select the appropriate users. If an assigned user does not already have a matching DNSimple user with the same email address, a DNSimple user will be provisioned for them and the DNSimple account admin will receive a notification that they have been added. If an assigned user already exists, they will need to link their Okta user on the User Settings page within DNSimple.
```

File: content/articles/okta-identity-provider.markdown [hunk 21]
```diff
@@ -0,0 +1,72 @@
...
+
+<info>
+Our integration works with groups! Leverage them to simplify your team access management within Okta.
+</info>
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:61]:
```suggestion
```

File: content/articles/okta-identity-provider.markdown [hunk 22]
```diff
@@ -0,0 +1,72 @@
...
+### Access control
+
+Now, you can set the level of DNSimple access for each member by visiting the Account > Members tab in your DNSimple account. For the full details, visit [Domain Access Control](/articles/domain-access-control/).
+
+If you experience any issues or have any questions, please reach out at support@dnsimple.com.
+
+## Remove the configuration
+
+1. Go to the <label>Account</label> page, and click the <label>Single Sign-On</label> tab.
+1. Click <label>Revoke</label> next to the SSO Workspace you wish to remove.
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:72]:
```suggestion
1. Click <label>Revoke</label> next to the SSO Workspace you wish to remove.

### Linking a DNSimple user to an Okta user

When a DNSimple user exists in a DNSimple account before _SSO_ is enabled, they will be required to link their Okta identity to verify ownership of both DNSimple users. To link a DNSimple user to an Okta user:

1. Go to the User Settings page.
1. In the Identities card, click <label>Add</label> next to the Okta identity provider.
1. Authenticate your Okta account.
1. You can now log in to DNSimple using the linked Okta identity.
1. You can remove the link by clicking <label>Remove</label> on the User Settings page.

```

File: content/articles/okta-identity-provider.markdown [hunk 23]
```diff
@@ -0,0 +1,79 @@
...
+
+### Linking a DNSimple user to an Okta user
+
+When a DNSimple user exists in a DNSimple account before _SSO_ is enabled, they will be required to link their Okta identity to verify ownership of both DNSimple users. To link a DNSimple user to an Okta user:
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:72]:
```suggestion
When a DNSimple user exists in a DNSimple account _before_ SSO is enabled, they will be required to link their Okta identity to verify ownership of both DNSimple users. To link a DNSimple user to an Okta user:
```

File: content/articles/okta-identity-provider.markdown [hunk 24]
```diff
@@ -0,0 +1,79 @@
...
+- Have administrator access to a DNSimple account
+- Have the Okta integration feature enabled on your DNSimple account
+
+## Supported Features
+
+- **SP-initiated Single Sign-On (SSO)**: This authentication flow occurs when the user attempts to log in to DNSimple from Okta.
+- **Import Users**: Manage who can access your DNSimple account by assigning users to the Okta application
+- **Single Log Out (SLO)**: Users can log out of their DNSimple session by logging out of their Okta session.
+
+## Configure Okta with DNSimple
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:33]:
```suggestion
## Link an Okta organization to a DNSimple account
```

File: content/articles/okta-identity-provider.markdown [hunk 25]
```diff
@@ -0,0 +1,79 @@
...
+### Add the DNSimple App to Okta
+
+1. Login to your organization's Okta marketplace.
+1. Search for DNSimple, and then click Add.
+1. Enter an Application Label in General Settings.
+1. Click Done.
+1. Assign yourself to the application.
+1. Take note of your Client ID, Client Secret, and Okta Domain as you'll need it in the next step.
+
+### Configure Okta for SSO in DNSimple
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:46]:
```suggestion
### Configure DNSimple for Okta SSO
```

File: content/articles/okta-identity-provider.markdown [hunk 26]
```diff
@@ -0,0 +1,79 @@
...
+
+In order to proceed with configuring login with SSO through Okta, you must:
+
+- Have administrator access to an Okta organization
+- Have administrator access to a DNSimple account
+- Have the Okta integration feature enabled on your DNSimple account
+
+## Supported Features
+
+- **SP-initiated Single Sign-On (SSO)**: This authentication flow occurs when the user attempts to log in to DNSimple from Okta.
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:29]: Do we support `JIT (Just In Time) Provisioning`?

File: content/articles/okta-identity-provider.markdown [hunk 27]
```diff
@@ -0,0 +1,79 @@
...
+1. Click <label>Authorize with Okta</label> to authorize the Workspace. You'll need to log in to the Okta Workspace with access to the `okta.eventHooks.manage`  and `okta.eventHooks.read` scopes.
+1. After logging in via Okta, you will arrive back on the DNSimple Single Sign-On page with Okta SSO enabled for the account.
+1. Repeat the process for each account where you want Okta SSO enabled.
+
+If the Okta app is no longer linked (e.g. the access token is revoked), you can re-link the Okta application using the "Reauthorize with Okta" button.
+
+### Assign People or Groups
+
+To give people access to DNSimple, click "Assignments" in under the configured DNSimple app in your Okta dashboard. Then, click "Assign" and select the appropriate users. If an assigned user does not already have a matching DNSimple user with the same email address, a DNSimple user will be provisioned for them and the DNSimple account admin will receive a notification that they have been added. If an assigned user already exists, they will need to link their Okta user on the User Settings page within DNSimple.
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:57]:
```suggestion
To give people access to DNSimple, click "Assignments" in under the configured DNSimple app in your Okta dashboard. Then, click "Assign" and select the appropriate users. If an assigned user does not already have a matching DNSimple user with the same email address, a DNSimple user will be provisioned for them and the DNSimple account admin will receive a notification that they have been added. If an assigned user already exists, they will need to link their Okta identity on the User Settings page within DNSimple.
```

File: content/articles/okta-identity-provider.markdown [hunk 28]
```diff
@@ -0,0 +1,79 @@
...
+Now, you can set the level of DNSimple access for each member by visiting the Account > Members tab in your DNSimple account. For the full details, visit [Domain Access Control](/articles/domain-access-control/).
+
+If you experience any issues or have any questions, please reach out at support@dnsimple.com.
+
+## Remove the configuration
+
+1. Go to the <label>Account</label> page, and click the <label>Single Sign-On</label> tab.
+1. Click <label>Revoke</label> next to the SSO Workspace you wish to remove.
+
+## Linking a DNSimple user to an Okta user
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:70]:
```suggestion
## Linking a DNSimple user to an Okta identity
```

File: content/articles/okta-identity-provider.markdown [hunk 29]
```diff
@@ -0,0 +1,79 @@
...
+
+When a DNSimple user exists in a DNSimple account _before_ SSO is enabled, they will be required to link their Okta identity to verify ownership of both DNSimple users. To link a DNSimple user to an Okta user:
```
Comment (dallasread) [inline content/articles/okta-identity-provider.markdown:72]:
```suggestion
When a DNSimple user exists in a DNSimple account _before_ SSO is enabled, they will be required to link their Okta identity to verify ownership of both DNSimple users. To link a DNSimple user to an Okta identity:
```

File: content/articles/okta-identity-provider.markdown (hunk 26 above)
Comment (lokst) [reply content/articles/okta-identity-provider.markdown:29]: I would think we don't currently? As the provisioning of a DNSimple account is done during app assignment in Okta

File: content/articles/okta-identity-provider.markdown [hunk 30]
```diff
@@ -0,0 +1,97 @@
...
+1. Take note of your Client ID, Client Secret, and Okta Domain as you'll need it in the next step.
+
+### Configure DNSimple for Okta SSO
+
+1. At DNSimple, go to the <label>Account</label> page, and click the <label>Single Sign-On</label> tab.
+1. Click <label>Authorize with Okta</label> to authorize the Workspace. You'll need to log in to the Okta Workspace with access to the `okta.eventHooks.manage`  and `okta.eventHooks.read` scopes.
+1. After logging in via Okta, you will arrive back on the DNSimple Single Sign-On page with Okta SSO enabled for the account.
+1. Repeat the process for each account where you want Okta SSO enabled.
+
+If the Okta app is no longer linked (e.g. the access token is revoked), you can re-link the Okta application using the "Reauthorize with Okta" button.
```
Comment (lokst) [inline content/articles/okta-identity-provider.markdown:61]: What do you think of this example `(e.g. the event hook is removed)` ?

File: content/articles/okta-identity-provider.markdown [hunk 31]
```diff
@@ -0,0 +1,97 @@
...
+
+### Assign People or Groups
+
+To give people access to DNSimple, click "Assignments" in under the configured DNSimple app in your Okta dashboard. Then, click "Assign" and select the appropriate users. 
+
+If an assigned user does not already have a matching DNSimple user with the same email address, a DNSimple user will be provisioned for them and the DNSimple account admin will receive a notification that they have been added.
+
+If an assigned user already exists, they will need to [link their DNSimple user to their Okta identity](#linking-a-dnsimple-user-to-an-okta-identity) on the User Settings page within DNSimple.
```
Comment (lokst) [inline content/articles/okta-identity-provider.markdown:69]: What do you think about

```
If an assigned user already exists, they will need to [link their DNSimple user to their Okta identity](#linking-a-dnsimple-user-to-an-okta-identity) on the User Settings page within DNSimple, before they can log in to DNSimple with their Okta identity.
```

File: content/articles/okta-identity-provider.markdown [hunk 32]
```diff
@@ -0,0 +1,97 @@
...
+If you experience any issues or have any questions, please reach out at support@dnsimple.com.
+
+## Unlinking an Okta organization from a DNSimple account
+
+1. Go to the <label>Account</label> page, and click the <label>Single Sign-On</label> tab.
+1. Click <label>Revoke</label> next to the SSO Workspace you wish to remove.
+
+## Linking a DNSimple user to an Okta identity
+
+When a DNSimple user exists in a DNSimple account _before_ SSO is enabled, they will be required to link their Okta identity to verify ownership of both DNSimple users. To link a DNSimple user to an Okta identity:
```
Comment (lokst) [inline content/articles/okta-identity-provider.markdown:86]: How about `to verify ownership of both DNSimple and Okta user accounts` ?

**Review by lokst — APPROVED** [review]
This looks good to me! I added some comments for consideration 🙂

File: content/articles/okta-identity-provider.markdown (hunk 30 above)
Comment (kojdm) [reply content/articles/okta-identity-provider.markdown:61]: Should we mention that an event hook is created for the Okta org in https://dev-02367705-admin.okta.com/admin/workflow/eventhooks for the first linked account, and to refrain from altering the hook's configuration (although "Reauthorize with Okta" takes care of this")?

**Review by kojdm — APPROVED** [review]
LGTM! 💯

**Pull request merged**